apps/workflow-server/gcc-workflow-server/src/main/java/com/coremedia/labs/translation/gcc/workflow/GlobalLinkAction.java

```diff
@@ -103,12 +123,9 @@
           .registerTypeAdapter(Content.class, new ContentObjectSerializer())
           .create();
 
-  private String skipVariable;
-  private String masterContentObjectsVariable;
-  private String remainingAutomaticRetriesVariable;
-  private String issuesVariable;
-  private String retryDelayTimerVariable;
+  private static final String CMS_RETRY_DELAY_SETTINGS_KEY = "cms-retry-delay";
 
+  private static final String DEFAULT_GCC_RETRY_DELAY_SETTINGS_KEY = "gcc-retry-delay";
```

Comment:
Would be nice to have a short Javadoc on these keys. The meaning is not obvious to me.

IIUC, the first one is only looked up in configuration properties, while the latter is just a default key (overridden by all subclasses) and theoretically looked up in both content and properties.

Reply:
They have comments in `gcc-workflow.properties`. User documentation for _cms-retry-delay_ is also given in administration.md. Does ist require an additional doc here?

Reply:
Looked at it again and... yes, added comments here, too.